sys/net/gnrc/netif/gnrc_netif.c

```diff
@@ -1305,9 +1382,11 @@ static void *_gnrc_netif_thread(void *args)
     gnrc_netif_release(netif);
 
     while (1) {
-        DEBUG("gnrc_netif: waiting for incoming messages\n");
-        msg_receive(&msg);
+        msg_t msg;
+        _process_events_await_msg(netif, &msg);
```

Comment:
Maybe add a comment on top, that `msg` will be set, when `_process_events_await_msg()` returns. It wasn't clear to me immediately.

Reply:
will do

Reply:
addressed